doc/troubleshooting.md

```diff
@@ -153,13 +153,22 @@ This alarm occurs if the FSU is enabled when installing MotoROS2.
 The MotoROS2 driver attempts to enable the *Speed Feedback* parameters, but is unable to change the required parameters due to FSU settings.
 You must temporarily disable the CRC check for the *Speed Feedback* update to complete.
 
-Touch `[RESET]` to clear the active alarm.
+Boot the controller into *Maintenance* mode by holding `{Main Menu}` on the keypad.
```

Comment:
After we've turned it off, right? Or does just holding `{Main Menu}` reboot the controller for me ? :)